src/Common/CommandArguments.php

```diff
@@ -58,25 +89,9 @@ public function rawArg($arg)
         return $this;
     }
 
-    /**
-     * Escape the provided value, unless it contains only alphanumeric
-     * plus a few other basic characters.
-     *
-     * @param string $value
-     *
-     * @return string
-     */
-    public static function escape($value)
```

Comment:
Removing a public method is a bc break -- didn't notice this before. To merge this PR into 3.x, we'd need to restore this method.

Reply:
Restored that method.